js/scripts.js

```diff
@@ -1122,6 +1148,31 @@ var plugin_formcreator = new function() {
       });
    }
 
+   this.changeActor = function(type, value) {
```

Comment:
factorization of :
plugin_formcreator_ChangeActorRequester
plugin_formcreator_ChangeActorWatcher
plugin_formcreator_ChangeActorASsigned